conf/machine/include/imx-base.inc

```diff
@@ -366,6 +366,7 @@ WKS_FILE_DEPENDS ?= " \
 "
 
 WKS_FILE_DEPENDS_append_mx8 = " imx-boot"
+WKS_FILE_DEPENDS_append_mx8m = " imx-boot"
```

Comment:
Can we drop line 368? why both?

Reply:
Above definition is kept for `mx8qm` and `mx8x` machines, which are using _imx-boot_ but do not have `mx8m` defined.

Since they are only targeted to build from NXP BSP - removing above line would remove the _imx-boot_ from the WIC path for them.